.github/workflows/CI.yml

```diff
@@ -0,0 +1,84 @@
+name: SwifterSwift
+
+on: 
+  push:
+    branches: 
+      - master
+  pull_request:
+    branches: 
+      - master
+
+jobs:
+  Darwin:
+    name: Darwin
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        xcode: ['/Applications/Xcode_11.3.app/Contents/Developer']
+    steps:
+    - uses: actions/checkout@v1
+    - name: Bundle Install
+      run: bundle install
+    - name: Brew Upgrade
+      run: |
+        brew update
+        brew outdated xctool || brew upgrade xctool
+        brew install swiftlint
+    - name: Test iOS
+      run: |
+        xcodebuild clean build test -project SwifterSwift.xcodeproj -scheme SwifterSwift-iOS -destination 'platform=iOS Simulator,name=iPhone 11' | XCPRETTY_JSON_FILE_OUTPUT="xcodebuild-ios.json" xcpretty -f `xcpretty-json-formatter`
+        bash <(curl -s https://codecov.io/bash) -cF ios -J 'SwifterSwift'
+      env:
```

Comment:
Can we use variables to make these `xcodebuild` commands more readable?

Reply:
Sure :))

Reply:
maybe `XCPRETTY_JSON_FILE_OUTPUT="xcodebuild-ios.json"` can go to `env:` part,

also run may be multiline such as
```
run: >
  xcodebuild clean build test 
    -project "$PROJECT"
    -scheme "$SCHEME"
    -destination "$DESTINATION" 
    | xcpretty -f "$(xcpretty-json-formatter)"
```

i took `>` from https://yaml-multiline.info/
also wrapped variables into quotes in case of parameter expansion with space